app/src/main/java/com/sopt/now/compose/screen/signin/SignInViewModel.kt

```diff
@@ -0,0 +1,29 @@
+package com.sopt.now.compose.screen.signin
+
+import androidx.lifecycle.ViewModel
+import com.sopt.now.compose.MyApplication
+import com.sopt.now.compose.screen.model.User
+
+class SignInViewModel : ViewModel() {
+
+
+    fun checkInvalidSignIn(id:String, pw:String): Boolean{
+        val user = getUSer()
+        when {
+            id.isBlank() -> return false
+            pw.isBlank() -> return false
+            id == user.id && pw == user.pw -> {
+                return true
+            }
+
+            else -> return false
+        }
+    }
```

Comment:
```suggestion
    fun checkInvalidSignIn(id:String, pw:String): Boolean = id == user.id && pw == user.pw 
```
그냥 이렇게 적을 수도 있을 것 같네요 ~